python/lsst/phosim/utils/phosim_repackager.py

```diff
@@ -41,10 +41,12 @@ def noao_section_keyword(bbox, flipx=False, flipy=False):
 class PhoSimRepackager:
     """
     Class to repackage phosim amplifier files into single sensor
-    MEFs with one HDU per amp.
+    MEFs with one HDU per amp, using LSE-400 for header conformity.
     """
     def __init__(self):
-        self.amp_info_records = list(list(PhosimMapper().camera)[0])
+        self.telcode = 'MC'
+        self.contrllr = 'H'
+        self.camera = LsstCam().getCamera()
```

Comment:
Please add the comments to explain the class attributes. For example, what is `self.telcode` or `self.contrllr`?

Reply:
Added comments.